lib/common/statusFileCreator.py

```diff
@@ -16,22 +16,82 @@ class StatusFileCreator(object):
     STATUS_TAG_SYNC_TASK_CONSUMER = "sync_task_consumer"
     STATUS_TAG_ASYNC_TASK_CONSUMER = "async_task_consumer"
 
+    # githubTasks.py
+    STATUS_TAG_GIT_RESET = "git_reset"
+    STATUS_TAG_GIT_FETCH = "git_fetch"
+    STATUS_TAG_GIT_CHECKOUT = "git_checkout"
+    STATUS_TAG_GIT_PULL = "git_pull"
+
+    # firefoxBuildTasks.py
+    STATUS_TAG_DOWNLOAD_NIGHTLY = "download_nightly"
+    STATUS_TAG_DEPLOY_FIREFOX_PKG = "deploy_firefox_pkg"
+
+    # hasalTasks.py
+    STATUS_TAG_GENERATE_HASAL_CONFIG = "generate_hasal_config"
+    STATUS_TAG_RUN_HASAL_RUNTEST_CMD = "run_hasal_runtest_cmd"
+
+    # runtest.py
+    STATUS_TAG_RUNTEST_CMD = "runtest_cmd"
+
+    # common tempate
+    STATUS_CODE_DESC_CMD_EXECUTED_SUCCESSFULLY = "Successfully complete"
+
+    STATUS_CODE_GENERAL_CMD_TEMPLATE = {100: "Prepare to execute the command",
+                                        800: "Failed on exeucting the command",
+                                        900: STATUS_CODE_DESC_CMD_EXECUTED_SUCCESSFULLY}
+
     STATUS_CODE_MAPPING = {STATUS_TAG_PULSE: {100: "Get message from pulse queue",
                                               200: "Put message into local queue",
-                                              900: "Successfully complete"},
+                                              900: STATUS_CODE_GENERAL_CMD_TEMPLATE},
```

Comment:
should be `STATUS_CODE_DESC_CMD_EXECUTED_SUCCESSFULLY`, not `STATUS_CODE_GENERAL_CMD_TEMPLATE` which is a `dict` object.